util/src/length_prefix_encoding/encoder.rs

```diff
@@ -92,6 +92,90 @@ pub struct WriteToIoReturn {
 }
 
 #[derive(Clone, Copy, Debug)]
+/// Module description goes here (skipped since it already exists in #479)
+///
+/// # Examples
+///
+///  Unfinished/proof-of-concept versions below - don't try this at home
+///
+/// **TODOs**:
+///
+/// * Actually write to a sink, if that's desirable/integration tests don't suffice
+/// * Consider splitting into multiple examples in that case
+/// * Resolve all TBD notes, fixup/polish, add references/links, etc.
+/// * rustfmt ignores docstrings, check [unstable option](https://github.com/rust-lang/rustfmt/issues/3348) or fix manually?
```

Comment:
We have a script for that  called `format_rust_code.sh`. If we could have rustfmt native support, that would be wonderful!

Reply:
It seems a bit out of scope, so maybe a separate issue would be warranted?

Reply:
@koraa I've created a [separate tracking issue](https://github.com/rosenpass/rosenpass/issues/562). With that, I'll consider this point resolved :)